src/components/CardPreview.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import {View} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import PropTypes from 'prop-types';
+import styles from '../styles/styles';
+import Text from './Text';
+import usePrivatePersonalDetails from '../hooks/usePrivatePersonalDetails';
+import ONYXKEYS from '../ONYXKEYS';
+import ExpensifyCardImage from '../../assets/images/expensify-card.svg';
+import variables from '../styles/variables';
+
+const propTypes = {
+    /** User's private personal details */
+    privatePersonalDetails: PropTypes.shape({
+        legalFirstName: PropTypes.string,
+        legalLastName: PropTypes.string,
+    }),
+    session: PropTypes.shape({
```

Comment:
Can we add a doc above this for completeness?

Reply:
Added